src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -176,7 +176,8 @@ static void InsertContentView(UIScrollView platformScrollView, IScrollView scrol
 			var contentContainer = new ContentView()
 			{
 				View = scrollView.PresentedContent,
-				Tag = ContentPanelTag
+				Tag = ContentPanelTag,
+				ClipsToBounds = true //Content should be clipped to the bounds of the ScrollView otherwise it will be visible outside the ScrollView but it can not be clicked causing problems.
```

Comment:
I'm pretty sure this is already fixed in another PR, but if not, I don't think ClipsToBounds improves the situation. We'd go from "things are visible, but cannot be tapped" to "things are invisible and cannot be tapped". That's actively worse for non-interactive controls.

Reply:
Removed, is the PR pending? I re tested the latest main yesterday and problem is still there.